README.md

```diff
@@ -61,7 +61,7 @@ This table lists the current status of the dependencies:
 | Push notifications <sup>(1) (2)</sup> | ✔️ Websocket | ⚠️ FCM<br>✔️ Websocket | ⚠️ FCM<br>✔️ Websocket |
 | Location provider                     | ✘           | ⚠️ Google Maps        | ⚠️️️ Google Maps        |
 
-<sup>(1)</sup> You may need to disable the system battery optimizations to receive Websocket-based  push notifications in background.<br>
+<sup>(1)</sup> You may need to disable the system battery optimizations to receive Websocket-based push notifications in background, as otherwise the app will be killed.<br>
```

Comment:
Technically this is not accurate. The app may not be killed from background and still have delayed notifications for many reasons. For example, by Android ignoring Molly wake up requests. I don't think telling the user about these technically nuances is important.

Reply:
Sounds good!